controllers/controllers/shared/index.go

```diff
@@ -27,3 +42,15 @@ func SetupIndexWithManager(mgr manager.Manager) error {
 
 	return nil
 }
+
+func setupServiceBindingIndex(mgr manager.Manager) error {
+	// Generate indexes for CFServiceBinding on field serviceBinding.Spec.AppRef.Name for efficient querying.
+	if err := mgr.GetFieldIndexer().IndexField(context.Background(), &servicesv1alpha1.CFServiceBinding{}, ServiceBindingAppGUIDIndex, func(rawObj client.Object) []string {
+		serviceBinding := rawObj.(*servicesv1alpha1.CFServiceBinding)
+		return []string{serviceBinding.Spec.AppRef.Name}
+	}); err != nil {
```

Comment:
I know you just copied this from the existing example, but this `err != nil` check is buried really deeply. It would be clearer to move the `err` assignment up one line and just have a simple `if err != nil` check on its own line.

Reply:
Done!